app_dart/micro_services/results_pipeline/analysis_options.yaml

```diff
@@ -0,0 +1,196 @@
+# Specify analysis options.
```

Comment:
nit: we should move to using pub.dev/packages/lints

Reply:
done